universe/Condition.cpp

```diff
@@ -9968,6 +9968,14 @@ unsigned int Not::GetCheckSum() const {
 ///////////////////////////////////////////////////////////
 // OrderedAlternativesOf
 ///////////////////////////////////////////////////////////
+void FCMoveContent(Condition::ObjectSet& from_set, Condition::ObjectSet& to_set) {
+    for (auto it = from_set.begin(); it != from_set.end(); ) {
+        to_set.push_back(*it);
+        *it = from_set.back();
+        from_set.pop_back();
+    }
+}
+
```

Comment:
Something like https://github.com/freeorion/freeorion/blob/master/universe/Condition.cpp#L423 doesn't work?

Reply:
it works and (Un)surprisingly also the move iterator does work now... must have been something else before